junit-runtime/src/main/scala/org/scalajs/junit/Reporter.scala

```diff
@@ -117,12 +117,14 @@ private[junit] final class Reporter(eventHandler: EventHandler,
   private def emitEvent(
     method: Option[String],
     status: Status,
-    throwable: OptionalThrowable = new OptionalThrowable
+    throwable: OptionalThrowable = new OptionalThrowable,
+    timeInSeconds: Double = 0.001
```

Comment:
That default value looks very suspicious. I think it should be an `Option[Double] = None`, if anything, and that `None` really translates to the constant `-1L`, as specified in the doc of `sbt.testing.Event`:
https://github.com/sbt/test-interface/blob/e0db1f531713ac7b66d3159513d2c044bc2cdca0/src/main/java/sbt/testing/Event.java#L37-L40

Reply:
Thanks!! Not only was it ugly and suspicious, it was also wrong (positive instead of negative).
I reworked it the way you suggested.
I also reworked the `throwable` parameter the same way and dropped the defaults for both: let'd be explicit.
This surfaced an issue: on JVM, time is reported for skipped tests (which `TestMetadata` calls `ignored`),
but on Scala.js it is not; this is why in `JUnitTest` I record duration as not populated for such tests.

Reply:
Alternatively, we can report duration of `0` in `Reporter.reportIgnored()`... Please advise ;)

Reply:
Our design is to emulate what the original JUnit does to the greatest extent possible. So if JUnit-on-JVM reports a duration for ignored tests, we should do the same, even if it's `0` as you suggest.